tests/functional/flakes/common.sh

```diff
@@ -71,6 +74,7 @@ createGitRepo() {
     rm -rf "$repo" "$repo".tmp
     mkdir -p "$repo"
 
+    # shellcheck disable=SC2086 # word splitting of extraArgs is intended
```

Comment:
maybe we do a `shift` after local repo="$1" and than we can just use "$@"

Reply:
I can replace `$extraArgs` with `$@` using the `shift`, but quoting `"$@"` breaks it:
```
    +(flakes.sh:34) createGitRepo /tmp/nix-test/flakes/flakes/flake1 ''
    +(./common.sh:71) local repo=/tmp/nix-test/flakes/flakes/flake1
    +(./common.sh:72) shift
    +(./common.sh:74) rm -rf /tmp/nix-test/flakes/flakes/flake1 /tmp/nix-test/flakes/flakes/flake1.tmp
    +(./common.sh:75) mkdir -p /tmp/nix-test/flakes/flakes/flake1
    +(./common.sh:78) git -C /tmp/nix-test/flakes/flakes/flake1 init ''
    fatal: cannot mkdir : No such file or directory
```

Reply:
Not super important. Anyway.